examples/darwin-framework-tool/commands/provider/OTAProviderDelegate.mm

```diff
@@ -72,35 +77,13 @@ - (void)handleQueryImageForNodeID:(NSNumber * _Nonnull)nodeID
     }
 
     _selectedCandidate.updateToken = [self generateUpdateToken];
+    NSLog(@"Query Image Status: %hhu", _queryImageStatus);
+    _selectedCandidate.status = @(_queryImageStatus);
 
-    if (params.requestorCanConsent.integerValue == 1) {
-        _selectedCandidate.status = @(MTROtaSoftwareUpdateProviderOTAQueryStatusUpdateAvailable);
-        _selectedCandidate.userConsentNeeded
-            = (_userConsentState == OTAProviderUserUnknown || _userConsentState == OTAProviderUserDenied) ? @(1) : @(0);
+    if (_userConsentNeeded) {
```

Comment:
Shouldn't this still take `params.requestorCanConsent` into account?

Reply:
Yes, indeed